employee.js

```diff
@@ -0,0 +1,46 @@
+const { DataTypes } = require('sequelize');
+const sequelize = require('./dbConnection');
+
+const Employee = sequelize.define('Employee', {
+  
+  employeeId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  joiningDate: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+},{
+    timestamps:false // to avoid all additional columns
+});
+
+module.exports = Employee;
+
+const Project = require('./project');
+const ProjectEmployee = require('./projectEmployee');
+
+Employee.belongsToMany(Project, {
+  through: ProjectEmployee,
+  foreignKey: 'employeeId',
+});
+
+Employee.hasMany(ProjectEmployee,{
+    foreignKey :'employeeId',
+})
+
+Project.belongsToMany(Employee, {
+  through: ProjectEmployee,
+  foreignKey: 'projectId',
+});
+
+Employee.sync({alter:false}).then(()=>{
```

Comment:
Question: why are we using `Employee.sync` rather than `sequelize.sync`

Reply:
I tried using Model.sync rather than syncing all at once but, now I have modified the code.